iohk-monitoring/src/Cardano/BM/Output/Aggregation.lhs

```diff
@@ -145,13 +145,13 @@ spawnDispatcher :: (Show a)
                 -> TBQ.TBQueue (Maybe (LogObject a))
                 -> Trace.Trace IO a
                 -> IO (Async.Async ())
-spawnDispatcher conf aggMap aggregationQueue trace0 = do
+spawnDispatcher conf aggMap aggregationQueue basetrace = do
```

Comment:
I was going to ask if you were going to revert the previous renaming (to avoid name clashes with `trace`) but these are probably better names anyway. :+1: